### PR TITLE
A brief about myStory.

### PR DESCRIPTION
**Pull request by mayankcool786**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2087)

### CODE REVIEW

**Comment by mayankcool786** [comment]
Please review this, @casey-collab

File: english/myStory/mayank.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Here I want to tell some of my life incidents which transformed me as a person.
```
Comment (casey-collab) [inline english/myStory/mayank.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/myStory/mayank.md (hunk 1 above)
Comment (mayankcool786) [inline english/myStory/mayank.md]: Done

**Comment by mayankcool786** [comment]
Done

File: english/myStory/mayank.md (hunk 1 above)
Comment (mayankcool786) [inline english/myStory/mayank.md]: Done @casey-collab

**Comment by mayankcool786** [comment]
Done @casey-collab

**Comment by casey-collab** [comment]
Looks great! :fire:

**Pull request merged**